UC-Berkeley-Extension/global
# Adding Pattern Lab

## Note
- See PR #4 for more complete implementation.

## Description
- Initial addition of Pattern Lab

## Testing Instructions
- Navigate to `global/web/themes/custom/bglobal/styleguide` dir
- run `npm install`
- run `npm start`
- Confirm that local pattern lab has been served.
<img width="500" alt="Screen Shot 2020-09-17 at 2 55 15 PM" src="https://user-images.githubusercontent.com/10488517/93521535-d1472c80-f8f5-11ea-8a79-db4731f6668b.png">
